tutorials/autotvm/tune_nnvm_arm.py

```diff
@@ -245,7 +253,7 @@ def tune_tasks(tasks,
     if os.path.exists(tmp_log_file):
         os.remove(tmp_log_file)
 
-    for i, tsk in enumerate(tasks):
+    for i, tsk in enumerate(reversed(tasks)):
```

Comment:
Why we use reversed here?

Reply:
Aha.. It does not matter a lot.
Typically the last few layers in a network have smaller search spaces, so we want to tune them first.
Because during the tuning we will collect all tuning log as history data to predict the next task.
So we want to first deal with the easy tasks and then hard tasks.

Later we will bring more sophisticated strategy to tune a bunch of tasks.